cime_config/machines/Depends.gnugpu.cmake

```diff
@@ -0,0 +1,24 @@
+list(APPEND MPAS_ICE_SHORTWAVE
```

Comment:
Following the patterns of the other changes in this PR you might want to rename the list to DISABLE_TREE_PTA_LIST or something like that

Reply:
And you might want to amend the commit message to include this flag

Reply:
The MPAS_ICE_SHORTWAVE name was already there. It's fine to rename it...
I don't know what you mean about amending commit message

Reply:
The commit message (or PR desc) does not mention the new compiler flag, -fno-tree-pta

Reply:
Oh I see -- that was already present for gnu. So adding this flag for GNU is not new, it's just we didn't have a gnugpu Depends file yet and therefore builds with gnugpu might also need that.  I can redo PR to exclude adding those flags.

Reply:
SO you think I should remove the flags to set `-fno-tree-pta`?

Reply:
No, just document that you are adding the flag in this PR

Reply:
ah, ok, thanks